### DISCUSSION TITLE
Country with search

### DISCUSSION THREAD

**Discussion by acetinick** [avo-hq/avo#1592] [Feature requests]
https://docs.avohq.io/2.0/fields/country.html

using the country field , works great, but ideally would be great to have a search true/false option here.

**Comment by adrianthedev** [comment]
So you'd like to have a multi-select like experience where you can choose multiple countries for a field?

**Comment by yorsant** [comment]
Having also a default value would be nice.

**Reply by adrianthedev** [reply]
Doesn't the `country` field support the [`default` option](https://docs.avohq.io/2.0/field-options.html#default-value)?

**Reply by yorsant** [reply]
Maybe I missed that. I'll check once I have some time.
